{
    "abilities": {
        "can_manage_common_responses?": false, 
        "can_manage_collaborators?": false, 
        "can_reopen?": false, 
        "can_ban_researcher?": false, 
        "can_create_severity?": false, 
        "can_assign_to_h1_triage?": false, 
        "can_agree_on_going_public?": false, 
        "assignable_team_member_groups": [], 
        "can_view_credential_account_details?": false, 
        "can_export?": false, 
        "can_create_retest?": false, 
        "can_assign_to_user?": false, 
        "can_use_common_responses?": false, 
        "can_hide_timeline?": false, 
        "can_be_manually_disclosed?": false, 
        "assignable_team_members": [], 
        "can_clone?": false, 
        "can_be_publicly_disclosed?": false, 
        "can_close_comments?": false, 
        "can_view_bounty_weights?": false, 
        "can_suggest_bounty_amount?": false, 
        "can_cancel_disclosure_request?": false, 
        "can_redact?": false, 
        "can_change_structured_scope?": false, 
        "can_post_internal_comments?": false, 
        "can_change_state?": false, 
        "can_change_weakness?": false, 
        "can_add_comment?": false, 
        "can_reassign_to_team?": false, 
        "can_change_title?": false, 
        "can_award_bounty?": false, 
        "can_award_swag?": false, 
        "can_close?": false, 
        "can_manage?": false
    }, 
    "comments_closed?": false, 
    "substate": "resolved", 
    "bug_reporter_agreed_on_going_public_at": null, 
    "voters": [
        "smit", 
        "dyabla", 
        "frrrrriend", 
        "louislane"
    ], 
    "facebook_team?": false, 
    "has_bounty?": false, 
    "rejected_anc_report_that_can_be_sent_back_to_anc_triagers?": false, 
    "original_report_id": null, 
    "id": 145409, 
    "can_view_team": true, 
    "team_member_agreed_on_going_public_at": "2016-06-27T20:51:05.053Z", 
    "activity_page_count": 1, 
    "activity_page_number": 1, 
    "title": "help.nextcloud.com: Known DoS condition (null pointer deref) in Nginx running", 
    "is_member_of_team?": null, 
    "vote_count": 4, 
    "summaries": [
        {
            "category": "team", 
            "can_create?": false, 
            "can_view?": true
        }, 
        {
            "category": "researcher", 
            "can_create?": false, 
            "can_view?": true
        }
    ], 
    "structured_scope": null, 
    "allow_singular_disclosure_at": "2016-07-27T20:51:05.149Z", 
    "state": "Closed", 
    "cve_ids": [], 
    "readable_substate": "Resolved", 
    "public": true, 
    "singular_disclosure_disabled": false, 
    "activities": [
        {
            "automated_response": true, 
            "created_at": "2016-06-17T14:10:20.613Z", 
            "is_internal": false, 
            "editable": false, 
            "updated_at": "2016-06-17T14:10:20.613Z", 
            "actor": {
                "url": "/nextcloud", 
                "profile": {
                    "name": "Nextcloud"
                }, 
                "ibb": false, 
                "profile_picture_urls": {
                    "medium": "https://profile-photos.hackerone-user-content.com/000/013/291/1d2ac8991616fcd3e3cdd567d02b7e70e20a3883_medium.png?1491410731"
                }
            }, 
            "team_handle": "nextcloud", 
            "message": "Thanks a lot for reporting this potential issue back to us!\n\nOur security team will take a look at this issue as soon as possible. We will reply to your report within 72 hours, usually much faster. For obvious reasons we'd like to ask you to not disclose this issue to any other party.", 
            "markdown_message": "<p>Thanks a lot for reporting this potential issue back to us!</p>\n\n<p>Our security team will take a look at this issue as soon as possible. We will reply to your report within 72 hours, usually much faster. For obvious reasons we&#39;d like to ask you to not disclose this issue to any other party.</p>\n", 
            "type": "Activities::Comment", 
            "id": 1019207, 
            "genius_execution_id": null
        }, 
        {
            "automated_response": false, 
            "created_at": "2016-06-17T22:38:12.791Z", 
            "is_internal": false, 
            "editable": false, 
            "updated_at": "2016-06-17T22:38:12.791Z", 
            "actor": {
                "username": "lukasreschke", 
                "url": "/lukasreschke", 
                "profile_picture_urls": {
                    "medium": "https://profile-photos.hackerone-user-content.com/000/037/891/0ddfc05411893988a0d2794cd70d5dc221729713_medium.png?1438782834"
                }, 
                "hackerone_triager": false
            }, 
            "team_handle": "nextcloud", 
            "message": "Thanks a lot for your report, a quick check on the system seems indeed to confirm that this runs 1.10.0:\n\n```\nroot@help-app:/var/www/discourse# nginx -v\nnginx version: nginx/1.10.0\n```\n\nIt seems that the next version of the Discourse docker file changes this to 1.10.1. I'll escalate this internally and get back to you once we have updated the container. ", 
            "markdown_message": "<p>Thanks a lot for your report, a quick check on the system seems indeed to confirm that this runs 1.10.0:</p>\n<pre class=\"highlight plaintext\"><code>root@help-app:/var/www/discourse# nginx -v\nnginx version: nginx/1.10.0\n</code></pre>\n<p>It seems that the next version of the Discourse docker file changes this to 1.10.1. I&#39;ll escalate this internally and get back to you once we have updated the container. </p>\n", 
            "type": "Activities::BugTriaged", 
            "id": 1020382, 
            "genius_execution_id": null
        }, 
        {
            "automated_response": false, 
            "created_at": "2016-06-27T20:46:37.567Z", 
            "is_internal": false, 
            "editable": false, 
            "updated_at": "2016-06-27T20:46:37.567Z", 
            "actor": {
                "username": "lukasreschke", 
                "url": "/lukasreschke", 
                "profile_picture_urls": {
                    "medium": "https://profile-photos.hackerone-user-content.com/000/037/891/0ddfc05411893988a0d2794cd70d5dc221729713_medium.png?1438782834"
                }, 
                "hackerone_triager": false
            }, 
            "additional_data": {
                "new_title": "help.nextcloud.com: Known DoS condition (null pointer deref) in Nginx running", 
                "old_title": "Help subsite has known DoS condition (null pointer deref) in Nginx running"
            }, 
            "team_handle": "nextcloud", 
            "message": "", 
            "markdown_message": "", 
            "type": "Activities::ReportTitleUpdated", 
            "id": 1041652, 
            "genius_execution_id": null
        }, 
        {
            "automated_response": false, 
            "reporter": {
                "username": "shoveller", 
                "url": "/shoveller"
            }, 
            "created_at": "2016-06-27T20:50:35.384Z", 
            "is_internal": false, 
            "editable": false, 
            "updated_at": "2016-06-27T20:58:55.729Z", 
            "actor": {
                "username": "lukasreschke", 
                "url": "/lukasreschke", 
                "profile_picture_urls": {
                    "medium": "https://profile-photos.hackerone-user-content.com/000/037/891/0ddfc05411893988a0d2794cd70d5dc221729713_medium.png?1438782834"
                }, 
                "hackerone_triager": false
            }, 
            "team_handle": "nextcloud", 
            "message": "Thanks a lot for your report. This has been fixed now by updating the used Docker container for Discourse.\n\nDiscourse was shipping a vulnerable version of Nginx until container version 1.3.4 (https://github.com/discourse/discourse_docker/commit/e8b9677b79e06b8864d87407b77e7800dc6c10f5) . We have fixed this by updating the container as described at https://meta.discourse.org/t/how-do-i-update-my-docker-image-to-latest/23325\n\n```\ncd /var/discourse\ngit pull\n./launcher rebuild app\n```\n\nA quick check verifies that we're indeed running 1.10.1 now:\n\n```\nroot@help-app:/var/www/discourse# nginx -v\nnginx version: nginx/1.10.1\n```\n\nThanks a lot for reporting this to us!\n\n", 
            "markdown_message": "<p>Thanks a lot for your report. This has been fixed now by updating the used Docker container for Discourse.</p>\n\n<p>Discourse was shipping a vulnerable version of Nginx until container version 1.3.4 (<a title=\"https://github.com/discourse/discourse_docker/commit/e8b9677b79e06b8864d87407b77e7800dc6c10f5\" href=\"/redirect?signature=80d998da724461c77a06c2598b9f6e207c540c13&amp;url=https%3A%2F%2Fgithub.com%2Fdiscourse%2Fdiscourse_docker%2Fcommit%2Fe8b9677b79e06b8864d87407b77e7800dc6c10f5\" target=\"_blank\" rel=\"nofollow noopener noreferrer\"><span>https://github.com/discourse/discourse_docker/commit/e8b9677b79e06b8864d87407b77e7800dc6c10f5</span><i class=\"icon-external-link\"></i></a>) . We have fixed this by updating the container as described at <a title=\"https://meta.discourse.org/t/how-do-i-update-my-docker-image-to-latest/23325\" href=\"/redirect?signature=a7d3b3762087ec1818a70b7717861cd318c2b792&amp;url=https%3A%2F%2Fmeta.discourse.org%2Ft%2Fhow-do-i-update-my-docker-image-to-latest%2F23325\" target=\"_blank\" rel=\"nofollow noopener noreferrer\"><span>https://meta.discourse.org/t/how-do-i-update-my-docker-image-to-latest/23325</span><i class=\"icon-external-link\"></i></a></p>\n<pre class=\"highlight plaintext\"><code>cd /var/discourse\ngit pull\n./launcher rebuild app\n</code></pre>\n<p>A quick check verifies that we&#39;re indeed running 1.10.1 now:</p>\n<pre class=\"highlight plaintext\"><code>root@help-app:/var/www/discourse# nginx -v\nnginx version: nginx/1.10.1\n</code></pre>\n<p>Thanks a lot for reporting this to us!</p>\n", 
            "type": "Activities::BugResolved", 
            "id": 1041662, 
            "genius_execution_id": null
        }, 
        {
            "automated_response": false, 
            "created_at": "2016-06-27T20:50:59.071Z", 
            "is_internal": false, 
            "editable": false, 
            "updated_at": "2016-06-27T20:50:59.071Z", 
            "actor": {
                "url": "/nextcloud", 
                "profile": {
                    "name": "Nextcloud"
                }, 
                "ibb": false, 
                "profile_picture_urls": {
                    "medium": "https://profile-photos.hackerone-user-content.com/000/013/291/1d2ac8991616fcd3e3cdd567d02b7e70e20a3883_medium.png?1491410731"
                }
            }, 
            "team_handle": "nextcloud", 
            "message": "As per our current bug bounty brief our web services are excluded from monetary rewards.", 
            "markdown_message": "<p>As per our current bug bounty brief our web services are excluded from monetary rewards.</p>\n", 
            "type": "Activities::NotEligibleForBounty", 
            "id": 1041663, 
            "genius_execution_id": null
        }, 
        {
            "automated_response": false, 
            "created_at": "2016-06-27T20:51:05.078Z", 
            "is_internal": false, 
            "editable": false, 
            "updated_at": "2016-06-27T20:51:05.078Z", 
            "actor": {
                "username": "lukasreschke", 
                "url": "/lukasreschke", 
                "profile_picture_urls": {
                    "medium": "https://profile-photos.hackerone-user-content.com/000/037/891/0ddfc05411893988a0d2794cd70d5dc221729713_medium.png?1438782834"
                }, 
                "hackerone_triager": false
            }, 
            "team_handle": "nextcloud", 
            "first_to_agree": true, 
            "message": "", 
            "markdown_message": "", 
            "type": "Activities::AgreedOnGoingPublic", 
            "id": 1041664, 
            "genius_execution_id": null
        }, 
        {
            "automated_response": false, 
            "created_at": "2016-07-27T20:51:19.811Z", 
            "is_internal": false, 
            "editable": false, 
            "updated_at": "2016-07-27T20:51:19.811Z", 
            "actor": {
                "url": "/nextcloud", 
                "profile": {
                    "name": "Nextcloud"
                }, 
                "ibb": false, 
                "profile_picture_urls": {
                    "medium": "https://profile-photos.hackerone-user-content.com/000/013/291/1d2ac8991616fcd3e3cdd567d02b7e70e20a3883_medium.png?1491410731"
                }
            }, 
            "team_handle": "nextcloud", 
            "message": "", 
            "markdown_message": "", 
            "type": "Activities::ReportBecamePublic", 
            "id": 1091741, 
            "genius_execution_id": null
        }
    ], 
    "in_validation?": false, 
    "is_participant": false, 
    "singular_disclosure_allowed": true, 
    "reporter": {
        "username": "shoveller", 
        "hacker_mediation": false, 
        "hackerone_triager": false, 
        "disabled": false, 
        "url": "/shoveller", 
        "profile_picture_urls": {
            "small": "/assets/avatars/default-71a302d706457f3d3a31eb30fa3e73e6cf0b1d677b8fa218eaeaffd67ae97918.png"
        }, 
        "is_me?": false
    }, 
    "weakness": {
        "id": 48, 
        "name": "Denial of Service"
    }, 
    "is_external_bug": false, 
    "visibility": "full", 
    "allow_singular_disclosure_after": -74483998.24755917, 
    "disclosed_at": "2016-07-27T20:51:19.652Z", 
    "stage": 4, 
    "url": "https://hackerone.com/reports/145409", 
    "created_at": "2016-06-17T14:10:20.459Z", 
    "original_report_url": null, 
    "vulnerability_information_html": "<p>The <a title=\"https://help.nextcloud.com\" href=\"/redirect?signature=d11c64e882d41ab9d2e58f956c21262e3add1f02&amp;url=https%3A%2F%2Fhelp.nextcloud.com\" target=\"_blank\" rel=\"nofollow noopener noreferrer\"><span>https://help.nextcloud.com</span><i class=\"icon-external-link\"></i></a> sub-site is running Nginx/1.10.0 which is vuln to a known issue (CVE-2016-4450) which allows a remote malformed HTTP request to cause the Nginx process to crash.</p>\n\n<p>DoS testing is mentioned as not requested, but if you know of an issue give it a go .. </p>\n\n<p>You can determine the version running by requesting the IP of the site and getting the HTTP 301, eg: <a title=\"https://88.198.160.135\" href=\"/redirect?signature=e111a0f42c2bb74e8fa8a0790a331ef256e305a9&amp;url=https%3A%2F%2F88.198.160.135\" target=\"_blank\" rel=\"nofollow noopener noreferrer\"><span>https://88.198.160.135</span><i class=\"icon-external-link\"></i></a></p>\n\n<p><a title=\"https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2016-4450\" href=\"/redirect?signature=db6644f9d4ef6a652bde6164aee98098c4f66fef&amp;url=https%3A%2F%2Fweb.nvd.nist.gov%2Fview%2Fvuln%2Fdetail%3FvulnId%3DCVE-2016-4450\" target=\"_blank\" rel=\"nofollow noopener noreferrer\"><span>https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2016-4450</span><i class=\"icon-external-link\"></i></a></p>\n", 
    "vulnerability_information": "The https://help.nextcloud.com sub-site is running Nginx/1.10.0 which is vuln to a known issue (CVE-2016-4450) which allows a remote malformed HTTP request to cause the Nginx process to crash.\n\nDoS testing is mentioned as not requested, but if you know of an issue give it a go .. \n\nYou can determine the version running by requesting the IP of the site and getting the HTTP 301, eg: https://88.198.160.135\n\nhttps://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2016-4450", 
    "team_private?": false, 
    "team": {
        "profile": {
            "website": "https://nextcloud.com", 
            "about": "Access, share and protect your files, calendars, contacts, communication & more at home and in your enterprise.", 
            "twitter_handle": "nextclouders", 
            "name": "Nextcloud"
        }, 
        "handle": "nextcloud", 
        "url": "https://hackerone.com/nextcloud", 
        "state": "public_mode", 
        "profile_picture_urls": {
            "small": "https://profile-photos.hackerone-user-content.com/000/013/291/5d33b6e08fad356e1743fd899fe7d6dda9971209_small.png?1491410731", 
            "medium": "https://profile-photos.hackerone-user-content.com/000/013/291/1d2ac8991616fcd3e3cdd567d02b7e70e20a3883_medium.png?1491410731"
        }, 
        "awards_miles": false, 
        "permissions": [], 
        "id": 13291, 
        "default_currency": "usd"
    }, 
    "is_published": false
}